.github/workflows/deployment-jdk-ea.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest, buildjet-8vcpu-ubuntu-2204-arm]
-        jdk: [22]
+        jdk: [23]
```

Comment:
shouldn't EA become 24 then?

Reply:
Not supported by the current workflow. I can re-add this, but takes 30 to 60 min again - need to switch to Oracle action etc.